src/components/Footer/Footer.module.css

```diff
@@ -3,19 +3,27 @@
   right: 0;
   bottom: 0;
   left: 0;
-  padding: 20px;
+  padding: 4px 20px;
   margin: 20px 20px 0 20px;
   background-color: var(--color-bg);
   text-align: center;
 }
 
 .footer {
-  height: 30px;
   width: 100%;
   display: flex;
   align-items: center;
   justify-content: space-between;
-  background-color: var(--color-table-header);
+  background-color: var(--color-bg);
```

Comment:
It already inherits the background color from the parent container, so this line is not needed.